index.html

```diff
@@ -176,46 +176,65 @@ <h1 class="skills--title">Language</h1>
             
             <div class="skills">
                 <img src="./img/square.png" alt="square" class="skills-img">
-                <h1 class="skills--title">Frameworks</h1>
+                <h2 class="skills--title">Frameworks</h2>
 
                 <div class="skill--list--container">
                     <ul class="skill--item2">
-                        <li class="skill--item2">Javascript</li>
-                        <li class="skill--item2">Ruby</li>
-                        <li class="skill--item2">HTML</li>
-                        <li class="skill--item2">CSS</li>
+                        <li class="skill--item2">bootstrap</li>
+                        <li class="skill--item2">Ruby on Rails</li>
+                        <li class="skill--item2">RSPec</li>
+                        <li class="skill--item2">CapyBara</li>
+                        <li class="skill--item2">Selenium</li>
                     </ul>
                 </div>
 
-                
-
+            
             </div>
             <div class="skills">
-                <h1 class="skills--title">Skills</h1>
+                <h2 class="skills--title">Skills</h2>
                 <img src="./img/circle.png" alt="square" class="skills-img">
 
                 <div class="skill--list--container">
                     <ul class="skill--item3">
-                        <li class="skill--item3">Javascript</li>
-                        <li class="skill--item3">Ruby</li>
-                        <li class="skill--item3">HTML</li>
-                        <li class="skill--item3">CSS</li>
+                        <li class="skill--item3">Codekit</li>
+                        <li class="skill--item3">Github</li>
+                        <li class="skill--item3">Codepen</li>
+                        <li class="skill--item3">Gitlab</li>
+                        <li class="skill--item3">Terminal</li>
                     </ul>
                     </div>
 
             </div>
-            
 
         </section>
-
     </section>
-    
 
+</div>
 
+<div><section class="contactme">
+    <h3 id="form--title">I'm always interested in hearing about new projects, so if you'd like to chat please get in</h3>
+    <form action="https://formspree.io/f/xeqnbagq" method="POST">
+        <input name="full_name" class="contact--input" id="input--name" type="text" maxlength="30" placeholder="Full Name" required>
```

Comment:
- [ ] Please avoid using double hyphens(`--`) in the id of an element. It won't go well with javascript. Instead, you can use camelcase or underscores.